src/core/system.c

```diff
@@ -1,6 +1,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <time.h>
```

Comment:
I just requested access for Travis CI in OpenTomb, so that these questions shouldn't arise anymore once a suitable build script exists. So I guess a proper cross-platform build system which allows to compile OpenTomb (but doesn't need to create deliverable executables in the first shot, as it's only for automated compile tests) has some sort of priority.